runtime/runtime-params-estimator/test-contract/src/lib.rs

```diff
@@ -636,6 +636,35 @@ pub unsafe fn data_receipt_10b_1000() {
     );
 }
 
+// Function to subtract the base from the `data_receipt_creation_config`. This method doesn't
+// have a callback on created promises so there is no data dependency.
+#[no_mangle]
+pub unsafe fn data_receipt_base_10b_1000() {
+    let buf = [0u8; 1000];
+    current_account_id(0);
+    let buf_len = register_len(0);
+    read_register(0, buf.as_ptr() as _);
+
+    let method_name = b"data_producer_10b";
+    let args = b"";
+    let mut ids = [0u64; 1000];
+    let amount = 0u128;
+    let gas = prepaid_gas();
+    for i in 0..1000 {
+        ids[i] = promise_create(
+            buf_len,
+            buf.as_ptr() as _,
```

Comment:
Does this just cast it to the type that the parameter expects?

Reply:
Yes.